### PR TITLE
Fix for bash's memory-leak

### PR DESCRIPTION
**Pull request by rodnymolina**
Memory leak is observed during the execution of scripts that make use of bash-arrays. In scenarios where the offending script is executed on a regular basis (e.g. fancontrol), the leaking process may end up consuming most of the system resources.

In this PR i'm replacing bash in all the contexts where it executes (both host and dockers). The official patch for this issue is here: https://ftp.gnu.org/gnu/bash/bash-4.3-patches/bash43-040

### CODE REVIEW

File: sonic-slave/Dockerfile [hunk 1]
```diff
@@ -227,6 +227,9 @@ RUN apt-get update && apt-get install -y \
         python3-requests \
         python3-pytest \
         python3-colorama
```
Comment (lguohan) [inline sonic-slave/Dockerfile]: please add \

**Comment by lguohan** [comment]
@rodnymolina , please fix build error.

File: sonic-slave/Dockerfile (hunk 1 above)
Comment (rodnymolina) [reply sonic-slave/Dockerfile]: yep, i just fixed it, this one was a last-minute issue while merging my patch with latest github code...

**Comment by lguohan** [comment]
@rodnymolina , still build failure, please investigate.

**Comment by rodnymolina** [comment]
Thanks Gouhan, i'm aware of it, i'm re-building a github image locally to verify.

**Pull request merged**